infrahub_sdk/client.py

```diff
@@ -1282,6 +1282,64 @@ async def get_diff_summary(
 
         return node_diffs
 
+    async def get_diff_tree(
+        self,
+        branch: str,
+        name: str | None = None,
+        from_time: datetime | None = None,
+        to_time: datetime | None = None,
+        timeout: int | None = None,
+        tracker: str | None = None,
+        raise_for_error: bool | None = None,
```

Comment:
Will try to take a closer look tomorrow but `raise_for_error` is deprecated in the execute_graphql method. So we should not add that anywhere new.